src/components/GuestRegister.vue

```diff
@@ -0,0 +1,54 @@
+<template>
+<div class="max-w-3xl m-auto pt-10">
```

Comment:
This won't work yet with the Rails API, the API will be expecting a password, so I either have to add it here or figure out a way to change that on the API